src/zope/preference/README.rst

```diff
@@ -234,9 +234,11 @@ that implements a default preferences provider that can be added as a unnamed
 utility for each site.
 
   >>> root
-  <zope.site.folder.Folder ...>
+  <...Folder ...>
```

Comment:
Maybe look at `root.__class__` instead for a more meaningful assertion.

Reply:
I found this part of the test disturbing and difficult to understand. I don't like it when variables just magically appear from thin air. The setup for `root` and `rsm` is very short, and there's only one doctest that uses it. What if we inline that setup here? Then the assertion isn't really need at all.

Reply:
+1 to put the setup into the doctest.